toolkit/themes/shared/design-system/config.js

```diff
@@ -135,26 +168,32 @@ function formatTokens({ mediaQuery, surface, args }) {
     return "";
   }
 
+  let layer = TOKENS_LAYERS_MAP[prop];
+
   dictionary.allTokens = dictionary.allProperties = tokens;
 
   let formattedVars = formatVariables({
     format: "css",
     dictionary,
     outputReferences: args.options.outputReferences,
     formatting: {
-      indentation: mediaQuery ? "    " : "  ",
+      indentation: mediaQuery ? "      " : "    ",
     },
   });
 
   // Weird spacing below is unfortunately necessary for formatting the built CSS.
   if (mediaQuery) {
     return `
+${NEST_MEDIA_QUERIES_COMMENT}
+@layer ${layer} {
   @media (${mediaQuery}) {
-${formattedVars}
+${generateBaseSelector(4, formattedVars, prop)}
```

Comment:
I dunno if we really need to make this a function 🤔  passing the number as an argument feels a bit weird, and we seem to be calculating the same space above on line 180 (`indentation: mediaQuery ? "    " : "  "`)`

What do you think about just leaning into using template literals here?

```js
  let layer = `tokens-${mediaQuery ?? "default"}`;

  if (mediaQuery) {
    return `
${NEST_MEDIA_QUERIES_COMMENT}
@layer ${layer} {
  @media (${mediaQuery}) {
    :root,
    :host(.anonymous-content-host) {
${formattedVars}
    }
  }
}
`;
  }

  return `@layer ${layer} {
  :root,
  :host(.anonymous-content-host) {
${formattedVars}
  }
}`;
```

Spacing might be a bit off after I copy/pasted, but this seems to work for me locally.

Reply:
hah whoops just looking at this again and realized it should be ``tokens-${mediaQuery ?? "foundation"}`...but you get the idea

Reply:
I like this way better than the function I wrote up, thanks for the suggestion!